configs/projects/_shared-pe-bolt-server.rb

```diff
@@ -74,6 +74,12 @@
 # R10k dependencies
 proj.component('rubygem-gettext-setup')
 
+# Building native extensions for the x25519 gem isn't supported on
+# el6 or 32-bit systems
+unless %w[el6 ubuntu16.04].include?(platform.dist)
```

Comment:
So it builds on amd64? we only build bolt-server for 64 bit ubuntu. So you can remove that here and only skip el6. So bolt wont support either arch on ubuntu 16.04? Seems like maybe there should be a way to differentiate between the two?